src/util/fields.py

```diff
@@ -182,7 +183,7 @@ def filter_value(key, value):
     return changed
 
 
-def remove_field(cr, model, fieldname, cascade=False, drop_column=True, skip_inherit=()):
+def remove_field(cr, model, fieldname, cascade=False, drop_column=True, skip_inherit=(), keep_attachments=False):
```

Comment:
`keep_attachments` sounds like if we were keeping _existing_ attachments
```suggestion
def remove_field(cr, model, fieldname, cascade=False, drop_column=True, skip_inherit=(), keep_as_attachments=False):
```